src/Compilers/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.csproj

```diff
@@ -15,10 +15,6 @@
     <PackageDescription>
       .NET Compiler Platform ("Roslyn") support for C#, Microsoft.CodeAnalysis.CSharp.dll.
     </PackageDescription>
-
-    <!-- Disable the nullable warnings when compiling for netcoreapp3.1 during our transition period -->
-    <DisableNullableWarnings Condition="'$(TargetFramework)' == 'netcoreapp3.1'">true</DisableNullableWarnings>
```

Comment:
Are there remaining places where we set `DisableNullableWarnings` to `true`?

Reply:
Not at this time. However it's likely that other projects are going to hit this when they do a similar conversion as the compiler. Hence I left it in for now.